website/layouts/partials/nav.html

```diff
@@ -0,0 +1,3 @@
+{{/* overriden not to render the navigation panel, the dummy <span> element
+prevents Antora JavaScript errors */}}
+<span class="nav-container"></span>
```

Comment:
```suggestion
```

Reply:
I jut saw the comment. How about:
```suggestion
<span class="nav-container" style="display:none;"></span>
```

Reply:
Should be fixed in 1ff7da935f039d9c23eb23bb17ce381261b8f05c